docs/09_Pages/Navigating_Pages/01_Internal_Links.md

```diff
@@ -34,4 +34,37 @@ You can use either `link` or `button_link` for the theme.
 
 ### Plain HTML Link
 
-`<a onclick="javascript:drupalgap_goto('node/123');">My Node Link</a>`
\ No newline at end of file
+`<a onclick="javascript:drupalgap_goto('node/123');">My Node Link</a>`
+
+### Theme A Button Link
+
+Add an attributes section to the button link and a class.
+
+```
+content['my_link'] = {
+  theme: 'link',
```

Comment:
@mkinnan Are you aware of the `button_link` theme type? It'll automatically make a jQM button for you. Did you still want me to merge this in?